developer-workflow/c-api.rst

```diff
@@ -106,6 +109,24 @@ Guidelines for expanding/changing the public API:
 - Make sure the ownership rules and lifetimes of all applicable struct
   fields, arguments and return values are well defined.
 
+- Functions returning ``PyObject *`` must return a valid pointer on success
```

Comment:
```suggestion
- Functions returning ``PyObject *`` must return a valid pointer on success,
```

While the SemBr makes the intent clear in the source, I was confused reading the rendered prose due to the ambiguous syntax and actually read it as (success and ``NULL``), (with an exception raised on error). A comma here clears that up.